### PR TITLE
enable hip30 epoch for testnet

### PR DESCRIPTION
**Pull request by diego1q2w**
## Issue

Setting the epoch activation for hip30 in testnet

### CODE REVIEW

File: internal/params/config.go [hunk 1]
```diff
@@ -116,7 +116,7 @@ var (
 		LeaderRotationExternalBeaconLeaders:    EpochTBD,
 		FeeCollectEpoch:                        big.NewInt(1296), // 2023-04-28 07:14:20+00:00
 		ValidatorCodeFixEpoch:                  big.NewInt(1296), // 2023-04-28 07:14:20+00:00
-		HIP30Epoch:                             EpochTBD,
+		HIP30Epoch:                             big.NewInt(2176), // 2023-10-10 10:00:00+00:00
```
Comment (sophoah) [inline internal/params/config.go:119]: big.NewInt(2176), // 2023-10-10 10:00:00+00:00 too early ? remember we need to sometime for the pre hip30 epoch for account migration to happen

**Comment by sophoah** [comment]
could you add activation epoch 30m gas as well

**Comment by sophoah** [comment]
yes same as hip30

File: internal/params/config.go (hunk 1 above)
Comment (sophoah) [reply internal/params/config.go]: can you update the comment with the correct date and time

**Review by sophoah — APPROVED** [review]

**Review by GheisMohammadi — APPROVED** [review]

**Pull request merged**